lib/rucio/client/baseclient.py

```diff
@@ -126,7 +126,7 @@ def __init__(self, rucio_host=None, auth_host=None, account=None, ca_cert=None,
         self.host = rucio_host
         self.list_hosts = []
         self.auth_host = auth_host
-        self.session = session()
+        self.session = Session()
```

Comment:
On line 318 this has to be changed as well.

Reply:
Done. I must have hit undo at some point...